### PR TITLE
update istio envoy tutorial to use AuthorizationPolicy

### PR DESCRIPTION
**Pull request by tjons**
### Why the changes in this PR are needed?

https://github.com/open-policy-agent/opa-envoy-plugin/pull/492 changes the `quick_start.yaml` file to use the Istio `AuthorizationPolicy` API instead of the `EnvoyFilter` API, per suggestion from @howardjohn (https://github.com/open-policy-agent/opa/issues/5911). 

### What are the changes in this PR?

Update the tutorial content for the Istio OPA integration to explain the changes made in https://github.com/open-policy-agent/opa-envoy-plugin/pull/492.

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
LGTM (as an Istio maintainer)

Thanks1

**Review by anderseknert — APPROVED** [review]
LGTM. Thanks!

File: docs/content/envoy-tutorial-istio.md [hunk 1]
```diff
@@ -5,22 +5,23 @@ weight: 11
...
 
 This tutorial requires Kubernetes 1.20 or later. To run the tutorial locally ensure you start a cluster with Kubernetes
 version 1.20+, we recommend using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube) or
 [KIND](https://kind.sigs.k8s.io/).
 
-The tutorial also requires Istio v1.8.0 or later. It assumes you have Istio deployed on top of Kubernetes.
+The tutorial also requires Istio v1.19.0 or later. It assumes you have Istio deployed on top of Kubernetes.
 See Istio's [Quick Start](https://istio.io/docs/setup/kubernetes/install/kubernetes/) page to get started.
 
+If you are using an earlier version of Istio (1.9+), you will have to customize the `AuthorizationPolicy` in the to use the
```
Comment (ashutosh-narkar) [inline docs/content/envoy-tutorial-istio.md:22]: Nit: `in the ...` ?

Comment (ashutosh-narkar) [reply docs/content/envoy-tutorial-istio.md:22]: I guess we're referring to the `quick_start.yaml` file here.

**Review by ashutosh-narkar — COMMENTED** [review]
LGTM. I guess we need to get https://github.com/open-policy-agent/opa-envoy-plugin/pull/492 first?

File: docs/content/envoy-tutorial-istio.md (hunk 1 above)
Comment (tjons) [reply docs/content/envoy-tutorial-istio.md]: Fixed!

**Comment by tjons** [comment]
Yes, the opa envoy plugin one should go first. Fixed the error you caught and squashed.

**Review by ashutosh-narkar — APPROVED** [review]

**Pull request merged**